scripts/src/public/services/offers.ts

```diff
@@ -44,12 +46,49 @@ type OfferTranslations = {
 	content: any;
 };
 
+type VersionRule = {
+	comparator: string;
+	data: VersionRuleData;
+};
+
+type VersionRuleData = {
+	[defaultKey: string]: string
+};
+
+const IMAGE_VERSION_RULES: VersionRule[] = [
+	{
+		comparator: ">=1.0.0",
+		data: {
+			"https://cdn.kinecosystem.com/thumbnails/offers/quiz_4.png": "https://s3.amazonaws.com/assets.kinecosystemtest.com/images/offer_placeholder.png",
+			"https://cdn.kinecosystem.com/thumbnails/offers/earn_offer3.png": "https://s3.amazonaws.com/assets.kinecosystemtest.com/images/offer_placeholder.png",
+			"https://cdn.kinecosystem.com/thumbnails/offers/future_poll.png": "https://s3.amazonaws.com/assets.kinecosystemtest.com/images/offer_placeholder.png",
+			"https://cdn.kinecosystem.com/thumbnails/offers/marketplace_poll.png": "https://s3.amazonaws.com/assets.kinecosystemtest.com/images/offer_placeholder.png",
+			"https://cdn.kinecosystem.com/thumbnails/offers/kik_poll.png": "https://s3.amazonaws.com/assets.kinecosystemtest.com/images/offer_placeholder.png",
+			"https://cdn.kinecosystem.com/thumbnails/offers/quiz_1.png": "https://s3.amazonaws.com/assets.kinecosystemtest.com/images/offer_placeholder.png",
+		}
```

Comment:
@doodyparizada @nitzantomer Change this according to the new images mapping.
I'm pretty sure we need to have clients move to `1.0.0` as the UI is a breaking change, but discuss/argue amongst yourselves and decide.

Note: Rules are evaluated and take precedence in order. I'll add a comment.

Reply:
We left this to you. There was some confusion of the current location of the images